cdap-ui/package.json

```diff
@@ -115,6 +115,7 @@
     "react-dropzone": "3.6.0",
     "react-file-download": "0.3.1",
     "react-helmet": "3.1.0",
+    "react-paginate": "^4.1.1",
```

Comment:
use specific version for this.

Also I believe you need to rebuild the dll. @ajainarayanan ?